packages/bridge/contracts/l1/L1GovernanceRelay.sol

```diff
@@ -69,7 +81,7 @@ contract L1GovernanceRelay {
       targetData
     );
 
-    IInbox(inbox).createRetryableTicket{value: msg.value}(
+    IInbox(inbox).createRetryableTicket{value: l1CallValue}(
```

Comment:
The refund addresses here are rewritten by the inbox. I would suggest revising if this is the intended behaviour

https://github.com/OffchainLabs/arbitrum/blob/800a05eb33abd9137a60cc989b42000e67028389/packages/arb-bridge-eth/contracts/bridge/Inbox.sol#L334-L343

Reply:
we will review this in a separate meeting